support/kind/kind.go

```diff
@@ -28,78 +29,65 @@ import (
 
 var kindVersion = "v0.11.0"
```

Comment:
Let's make this a value that can be changed.  Hardcoding it means the user will be stuck with that version.
So, maybe a new constructor function like
```go
func NewClusterWithVersion(clusterName, version string){}
```

Or, add a method on type `Cluster`

```go
type Cluster struct
   version string
}

func (c *Cluster) WithVersion(ver string) *Cluster {
    c.version = ver
    return c
}
```

Reply:
Also, `Cluster.version` should be initialized with the default version (i.e. `v0.11.0`) in case it's never set using the above.